net-mgmt/zabbix-agent/src/www/diag_logs_zabbixagent.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+$logfile = '/var/log/zabbix/zabbix_agentd.log';
+$logclog = false;
+
+require_once 'diag_logs_template.inc';
```

Comment:
probably should be integrated into MVC, so the log can be downloaded via the API.